website/docs/reference/project-collaboration-mode.md

```diff
@@ -2,52 +2,64 @@
+## Open collaboration mode
 
-Only admins and project members can submit change requests in a **protected** project. Other users can not submit change requests.
+All users of your Unleash instance can access the project and submit change requests. This is the default collaboration mode.
 
-## **Private** collaboration mode
 
-Private collaboration mode renders the project invisible to all **viewers** who are not project members. This means that **viewers** cannot see the project in the project list, nor can they access the project's features or locate the project anywhere within Unleash.
+## Protected collaboration mode
 
-**Editors**, **admins**, and users with **custom root roles** can still see private projects.
+All users of your Unleash instance can access the project but only Admins and project Members can submit change requests.
 
-### Change requests
 
-When you change a project's collaboration mode from open to protected, users who do not have the right permissions will lose the ability to create new change requests.
+## Private collaboration mode
 
-However, existing change requests created by these users will not be deleted. Any users with open change requests will still be able to cancel the change request, but they will **not** be able to update the change request.
+Only project Members, Admins, Editors, and users with [custom root roles](rbac.md#custom-root-roles) can access the project. Only Admins and project Members can submit change requests. 
 
-## Project collaboration mode setting
 
-The collaboration mode for a project can be set at the time of creation and modified at any subsequent time, found within the 'Enterprise Settings' section.
+|           | View project                                                                                    | Submit change requests     |
+|-----------|-------------------------------------------------------------------------------------------------|----------------------------|
+| Open      | All users                                                                                       | All users                  |
+| Protected | All users                                                                                       | Project Members and Admins |
+| Private   | Project Members, Admins, Editors, and users with [custom root roles](rbac.md#custom-root-roles) | Project Members and Admins |
 
-![Project creation form with a collaboration mode field and corresponding explanation.](/img/collaboration-mode.png)
+# Set project collaboration mode
 
-Since the collaboration mode is an integral feature of the project, the options to modify it can be found on the 'Project Settings' page.
+To [create a new project](projects.md#create-a-project) with a specific collaboration mode, do the following:
+1. In the Unleash Admin UI, go to **Projects** > **New project**.
+2. Enter **Project name**.
+3. Click **Open** to choose your collaboration mode.
+4. Click **Create project**.
```

Comment:
Here's a question for the style guide: do we use words like "click"? Click is exclusively a mouse gesture. People who navigate by keyboards or touch screens don't "click" things. I believe I've seen style guides that discourages use of mouse-specific terminology because it's not inclusive of other ways to navigate the website. Do you have any thoughts around this?

Reply:
I'm not sure I've ever come across a good term that was truly inclusive of all different kinds of devices/ navigation approaches. Do you know of one? For me, one that comes close is _select_, but I prefer to reserve it for dropdowns.

So I think it's okay to continue using _click_ for now. I think it remains a deeply ingrained term that is widely recognizable and understood by most users.

For general UI documentation stuff, I would stick to this [style guide](https://developers.google.com/style/ui-elements), unless we find that it doesn't work well for us.

Reply:
Nah, I haven't heard any good ones. "Interact" comes up, but "Interact with the create project button" is both long, and ... can be kinda confusing.

But thanks for the link and for picking a style guide. I'm glad to have a definitive answer for this so I don't have to wonder anymore 😄